src/model/LabyrinthList.java

```diff
@@ -1,77 +0,0 @@
-package model;
-import java.io.File;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Scanner;
-
-public class LabyrinthList extends LabyrinthImpl {
-
-	private HashMap<Position, Integer> list;
-
-	@Override
-	public int getRowCount() {
-		return rows;
-	}
-
-	@Override
-	public int getColumnCount() {
-		return columns;
-	}
-
-	@Override
-	public boolean isFreeAt(Position p) {
-		return !list.containsKey(p);
-	}
-
-
-	@Override
-	public boolean isWallAt(Position p) {
-		return list.containsKey(p) && list.get(p) != startValue && list.get(p) != finishValue;
```

Comment:
This line basically searches for the same value three times in the same list :). Well, 3 is not such a big deal, but I'd usually rather sacrifice space than time, i.e. maybe store the found key (if any), and then compare it?